desktop-ui/presentation/presentation.hpp

```diff
@@ -53,7 +53,10 @@ struct Presentation : Window {
       MenuCheckItem pauseEmulation{&toolsMenu};
       MenuSeparator toolsMenuSeparatorB{&toolsMenu};
       MenuItem manifestViewerAction{&toolsMenu};
+      #if !defined(PLATFORM_MACOS)
+      // Cocoa hiro is missing the hex editor widget
       MenuItem memoryEditorAction{&toolsMenu};
+      #endif
```

Comment:
There's already a bunch of widgets that are platform specific:

https://github.com/higan-emu/ares/blob/c71f131b0cb2722c5c5d0591da87edeff816898d/hiro/components.hpp#L80-L85

Rather than adding a `PLATFORM_MACOS` check here, you should probably add a block like this in hiro to exclude the HexEdit widget on macOS, and add a check for `Hiro_HexEdit` here.

Reply:
That's true, I haven't considered it. Good idea.

Reply:
Looks like Hiro_HexEdit is defined even for macOS, despite not being implemented :S